.github/workflows/gradle-run-ui-tests.yml

```diff
@@ -12,11 +12,10 @@ concurrency:
 
 jobs:
   ui-tests:
-    if: ${{ contains(github.head_ref, 'qa/') || contains(github.ref_name, 'qa/')}}  # disable for as they are not yet completed
     runs-on: buildjet-8vcpu-ubuntu-2204
     strategy:
       matrix:
-        api-level: [30]
+        api-level: [29]
```

Comment:
```suggestion
        api-level: [30]
```

Reply:
api level 30 is acting weird I advice to keep it on 29 as I have tested on both apis and 30 time out 7/10 times where 29 never did